devops/openshift/database-backup/backup-build.yaml

```diff
@@ -73,4 +71,4 @@ parameters:
     displayName: FROM Image Tag
     description: Base image to build from.  Docker creds or Artificatory setup may be needed to alleviate docker rate-limiting
     required: true
-    value: artifacts.developer.gov.bc.ca/docker-remote/centos/postgresql-12-centos7:20200917-804ef01
+    value: docker.io/centos/postgresql-12-centos7:20200917-804ef01
```

Comment:
`artifacts.developer.gov.bc.ca/docker-remote` is not the same as `docker.io`?
Are we not supposed to be using the `artifacts` URL as much as possible (if not for everything)?

Reply:
Yes you are right, but in this case, the build.yaml has been copied from the https://github.com/BCDevOps/backup-container/blob/master/openshift/templates/backup/backup-build.yaml. And the base image is actually not uses this too instead it uses quay.io/fedora/postgresql-14:14, as per the suggestion from the bcgov developer who did the changes to the dockerfile, which supports postgres 14.

Reply:
@guru-aot does `artifacts.developer.gov.bc.ca/docker-remote/centos/postgresql-12-centos7:20200917-804ef01` work? as artifacts.developer.gov.bc.ca/docker-remote is just a proxy. 
Correct me if wrong.

IMO only if it does not work we should start thinking about docker.io IMO.